docs/dictionary-data.md

```diff
@@ -4,7 +4,400 @@
+    uniqueness.
+
+    How to create disambiguators is up to the lexicographer. For example,
+    for the Plains Cree homograph `yôtin`, the lexicographer might set the
+    slugs for the three entries to `yôtin@1` and `yôtin@2` and `yôtin@3`;
+    or to `yôtin@na` and `yôtin@ni` and `yôtin@v`; or following any other
+    scheme they desire, so long as the assignments remain relatively
+    stable.
+
+    Any homograph disambiguator should start with `@`, as there is code in
+    morphodict to redirect an attempt to access an invalid homograph
+    disambiguator like `/word/nîmiw@foo` into a search for `nîmiw`, whereas
+    something like `nîmiw-foo` will do a search for `nîmiw-foo` instead of
+    `nîmiw`. That way if disambiguators change due to adding and removing
+    entries/definitions, old links should still be useful to people.
+
+  - The `paradigm` field, an optional string, is the name of the paradigm
+    used to display the paradigm tables for the entry. This may be a static
+    or dynamic paradigm.
+
+    This field may have a null value, or be omitted entirely, if the entry
+    is indeclinable.
+
+  - The `analysis` field, an optional array, is the analysis of the
+    headword. This field is used to populate the linguistic breakdown popup
+    shown by the blue ℹ️ icon.
+
+    The required format is that of an entry from the list returned by
+    `lookup_lemma_with_affixes`, e.g., `[[], "nîmiw", ["+V", "+AI", "+Ind",
+    "+3Sg"]]`. The format is:
+
+      - Array of:
+
+          - Array of FST tag strings
+
+          - FST lemma string
+
+          - Array of FST tag strings
+
+    This field may have a null value, or be omitted entirely, if the entry
+    is unanalyzable.
+
+  - The `linguistInfo` field, an optional arbitrary JSON object, allows
+    extra presentation data to be stored in the database. Morphodict HTML
+    templates have access to this data for the purpose of displaying
+    additional data to the end user.
+
+    Morphodict does not use any of this data for its core
+    language-independent functionality.
+
+    It is recommended *not* to put any unused data in here that ‘might be
+    handy later,’ but only to add new things here when required as part of
+    a coordinated effort with the frontend code to add new user-facing
+    features.
+
+  - The `fstLemma` field, an optional string, is the FST lemma to use when
+    generating paradigm tables.
+
+    To be clear: the FST lemma is the thing that gets plugged into a
+    paradigm layout template to generate associated wordforms. For example,
+    if the head is `nîminâniwan` with FST analysis `nîmiw+V+AI+Ind+X`, then
+    the FST lemma is `nîmiw`. If the head is `nimîw` with the FST
+    analysis `nimîw+V+AI+Ind+3Sg`, then the FST lemma is `nimîw`, which is
+    the same as the head.
+
+    Normally the FST lemma is included as part of the `analysis`, so this
+    field may only be specified on entries without an analysis.
```

Comment:
Which do you mean?

* The `fstLemma` field should not be included if the `analysis` field is present.
* The `fstLemma` field may be omitted if the `analysis` field is present.